README.md

```diff
@@ -118,10 +124,32 @@ Restart vs code after `flutter upgrade`.
 
 ### ios
 
-#### clean script
+#### Build Resolution
 
-Nuclear Clean
+Most cases the following should be enough.
+
+```sh
+pod install;
+flutter clean;
+```
+
+Nuclear resolution
 
 ``` sh
-cd ios/; rm -rf ~/Library/Caches/CocoaPods Pods ~/Library/Developer/Xcode/DerivedData/*; pod deintegrate; pod setup; pod install;
+cd ios/; rm -rf ~/Library/Caches/CocoaPods Pods ~/Library/Developer/Xcode/DerivedData/*; gem install cocoapods --user-install; pod deintegrate; pod setup; pod install;
```

Comment:
I don't agree with the addition of `gem install cocoapods --user-install;` here. Perhaps I don't understand what the purpose of the overall command is but updating cocoapods is not really part of a clean and has system wide side effects. I think it should be separate and have an explanation of what it is doing and why it might help.

More generally, I personally think the Build Resolution section could be moved somewhere, perhaps a Wiki page or another markdown document in the repo, as it's not specific to this project and is a more general set of "commands that may be helpful for Flutter projects build resolution". I'm not meaning to pick on this section, there are other parts of the README that I would also say this of. I'd be happy to approve this PR and make a separate issue to clean up the README if that's preferable. 

Thanks!